docs/source/introduction.rst

```diff
@@ -170,36 +201,35 @@ Alternatively pymgclient can be installed by using :file:`setup.py`::
 Running the test suite
 ######################
 
-If pymgclient is installed from downloaded source, you can run the test suite
-to verify it is working correctly. From the source directory, you can run::
+If pymgclient is installed from downloaded source, you can run the test suite to
+verify it is working correctly. From the source directory, you can run::
```

Comment:
```suggestion
verify it is working correctly. From the source directory, you can run:
```

Reply:
I think this is necessary to format the following code block properly. If you check it, before every `  $ pip ....` line, there are double semicolons.